test/test-parse-expression.cpp

```diff
@@ -1442,6 +1442,21 @@ TEST_F(test_parse_expression, malformed_object_literal) {
                                 error_invalid_lone_literal_in_object_literal,
                                 where, offsets_matcher(p.locator, 1, 4))));
   }
+
+  {
+    test_parser p(u8"{`key`: 1};"_sv);
+    expression_ptr ast = p.parse_expression();
+    EXPECT_EQ(summarize(ast), "object(literal, ?)");
+    EXPECT_THAT(
+        p.errors(),
+        ElementsAre(
+            ERROR_TYPE_FIELD(error_unexpected_template_string, where,
+                             offsets_matcher(p.locator, 6, 6)),
```

Comment:
Blocking: This assertion is saying that the error should occupy no characters (starting at character 6 but ending before character 6). Shouldn't the error span the template literal (starting at character but ending before character 6)?